augur/utils.py

```diff
@@ -253,24 +253,31 @@ def read_lat_longs(overrides=None, use_defaults=True):
 
 def read_colors(overrides=None, use_defaults=True):
     colors = {}
+    def add_line(line):
+        if line.startswith('#'):
+            return
+        fields = line.strip().split()
+        if len(fields) != 3:
+            print("WARNING: Color map file contained this invalid line: ", line)
```

Comment:
This prints quite a lot for me just because of blank lines in my `color.tsv`. Some are trailing blank lines at the end of the file, others are blanks to separate categories (which I think is useful). Perhaps change so it doesn't print if it's just a blank line?